AndroidSDKPush/src/main/java/com/leanplum/LeanplumPushService.java

```diff
@@ -201,39 +201,32 @@ public static void setDefaultCallbackClass(Class<? extends Activity> callbackCla
   }
 
   /**
-   * Sets an object used to customize the appearance of notifications. Call this from your
-   * Application class's onCreate method so that the customizer is set when your application starts
-   * in the background.
+   * Deprecated: please use{@link
+   * LeanplumPushService#setCustomizer(LeanplumPushNotificationCustomizer, boolean)}
    */
+  @SuppressWarnings("deprecation")
+  @Deprecated
   public static void setCustomizer(LeanplumPushNotificationCustomizer customizer) {
-    if (LeanplumPushService.notificationBuilderCustomizer != null) {
-      Log.e("You can only use one customizer for push notifications. Please remove " +
-          "LeanplumPushService.setCustomizer(LeanplumNotificationBuilderCustomizer) from onCreate" +
-          "method in your Application class if you do not want to support 2 lines of text on " +
-          "BigPicture style push notification.");
-      return;
-    }
-    LeanplumPushService.customizer = customizer;
+    setCustomizer(customizer, false);
+
```

Comment:
empty line

Reply:
Hm, I am wondering if we should deprecate it now after all?
It seems to me that this is the method that most clients would want to use now?

Should we also flip the default value to `setCustomizer(customizer, false)` because then clients would get benefit for 2 lines support, no?

Reply:
yeah, right